app/Http/Controllers/PermissionsController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Request;
+use App\UserType;
+use App\User;
+
+class PermissionsController extends Controller 
+{
+
+
+	public function getUser()
+	{
+		if(!Auth::check())
```

Comment:
Next time place something like this

```
function __construct(){
    $this->middleware('auth');
}
```

With it only authenticated users can acces the route and non authenticated users will go to login.